app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -249,9 +270,18 @@
       display: block;
       margin-top: govuk-spacing(3);
       @supports (display: grid) {
-        margin-top: 0;
+        margin-top: govuk-spacing(5);
         grid-area: 2 / 1 / 2 / 3;
       }
+
+      &:before {
+        content: "";
```

Comment:
What is this element for? It looks like it's putting a border between the two sections on mobile. This seems like an overly verbose way of doing that, can it not be achieved a simpler way?

Reply:
As its a list and using grid this was a solution, am open to suggestions